src/types.rs

```diff
@@ -15,9 +15,27 @@ pub struct Measurement {
     pub humidity: Humidity,
 }
 
+/// A combined raw temperature / humidity measurement.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct RawMeasurement {
+    /// The measured temperature (raw value).
+    pub temperature: u16,
+    /// The measured humidity (raw value).
+    pub humidity: u16,
+}
+
+impl Into<Measurement> for RawMeasurement {
+    fn into(self) -> Measurement {
```

Comment:
I'd prefer to implement `From` for`Measurement` and get the generic `Into` implementation.